src/pages/SettingsPage/UsersSettings/newServiceUser.jsx

```diff
@@ -0,0 +1,180 @@
+import { useEffect, useRef, useState } from 'react'
+import { toast } from 'react-toastify'
+import styled from 'styled-components'
+import { SelectButton } from 'primereact/selectbutton'
+
+import { Button, Divider, SaveButton, Section, Dialog, FormRow } from '@ynput/ayon-react-components'
+import ayonClient from '@/ayon'
+import ApiKeyManager from '@/components/ApiKeyManager'
+import useUserMutations from '@/containers/Feed/hooks/useUserMutations'
+import { useAddUserMutation } from '@queries/user/updateUser'
+import copyToClipboard from '@/helpers/copyToClipboard'
+import callbackOnKeyDown from '@/helpers/callbackOnKeyDown'
+
+import UserAccessGroupsForm from './UserAccessGroupsForm/UserAccessGroupsForm'
+import UserAttribForm from './UserAttribForm'
+import UserAccessForm from './UserAccessForm'
+
+const FormRowStyled = styled(FormRow)`
+  .label {
+    min-width: 160px;
+  }
+`
+const DividerSmallStyled = styled(Divider)`
+  margin: 8px 0;
+`
+const SubTitleStyled = styled.span`
+  margin-top: 16px;
+  margin-bottom: 0;
+`
+
+const NewServiceUser = ({ onHide, open, onSuccess, accessGroupsData }) => {
+  const {
+    password, setPassword,
+    passwordConfirm, setPasswordConfirm,
+    formData, setFormData,
+    addedUsers,
+    resetFormData,
+  } = useUserMutations({})
+  const [addUser, { isLoading: isCreatingUser }] = useAddUserMutation()
+  const usernameRef = useRef()
+
+  const attributes = ayonClient.getAttribsByScope('user')
+
+  const validateFormData = (formData) => {
+    if (!formData.Username) {
+      return 'Login name must be provided';
+    }
+
+    return null;
+  }
+
+  const preparePayload = (formData, attributes, password) => {
+    const payload = {
+      data: {
+        isService: true,
+        defaultAccessGroups: formData.defaultAccessGroups || [],
+        accessGroups: formData.accessGroups || {},
+      },
+      attrib: {},
+      name: formData.Username,
+      password: password ? password : undefined,
```

Comment:
I think most of these fields can be removed now as they don't apply to service users?

Password needs to be apiKey instead.

<img width="367" alt="image" src="https://github.com/user-attachments/assets/2757ecaf-405e-4bcf-b528-c960f296c66e">